package.json

```diff
@@ -72,15 +73,15 @@
     "react-toggle": "^4.1.3",
     "redux-oidc": "^4.0.0-beta1",
     "skulpt": "^1.2.0",
-    "three": "^0.140.0",
+    "three": "0.169.0",
     "ts-pnp": "1.2.0",
+    "url": "^0.11.4",
     "watch": "^1.0.2",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
     "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
```

Comment:
🪓 get rid of the `build.js` script too then

Reply:
Ah okay, I will get rid now 🗑️